### PR TITLE
[grafana] RBAC rules for alerting sidecar

### PR DESCRIPTION
**Pull request by juris**
If you enable just an alerting sidecar, it won't work as there are no RBAC rules for it.

### CODE REVIEW

**Review by zanhsieh — CHANGES_REQUESTED** [review]
@juris 
Thanks for the contribution. Please bump the version in Chart.yaml otherwise CI won't allow to merge.

**Comment by juris** [comment]
> @juris Thanks for the contribution. Please bump the version in Chart.yaml otherwise CI won't allow to merge.

Bumped to the next one. Cheers!

File: charts/grafana/templates/clusterrole.yaml [hunk 1]
```diff
@@ -9,9 +9,9 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
   name: {{ include "grafana.fullname" . }}-clusterrole
-{{- if or .Values.sidecar.dashboards.enabled (or .Values.rbac.extraClusterRoleRules (or .Values.sidecar.datasources.enabled .Values.sidecar.plugins.enabled)) }}
+{{- if or .Values.sidecar.dashboards.enabled (or .Values.rbac.extraClusterRoleRules (or .Values.sidecar.datasources.enabled (or .Values.sidecar.plugins.enabled .Values.sidecar.alerts.enabled))) }}
```
Comment (zanhsieh) [inline charts/grafana/templates/clusterrole.yaml:12]:
```suggestion
{{- if or .Values.sidecar.dashboards.enabled .Values.rbac.extraClusterRoleRules .Values.sidecar.datasources.enabled .Values.sidecar.plugins.enabled .Values.sidecar.alerts.enabled }}
```

File: charts/grafana/templates/clusterrole.yaml [hunk 2]
```diff
@@ -9,9 +9,9 @@ metadata:
...
 rules:
-  {{- if or .Values.sidecar.dashboards.enabled (or .Values.sidecar.datasources.enabled .Values.sidecar.plugins.enabled) }}
+  {{- if or .Values.sidecar.dashboards.enabled (or .Values.sidecar.datasources.enabled (or .Values.sidecar.plugins.enabled .Values.sidecar.alerts.enabled)) }}
```
Comment (zanhsieh) [inline charts/grafana/templates/clusterrole.yaml:14]:
```suggestion
  {{- if or .Values.sidecar.dashboards.enabled .Values.sidecar.datasources.enabled .Values.sidecar.plugins.enabled .Values.sidecar.alerts.enabled }}

**Review by zanhsieh — CHANGES_REQUESTED** [review]

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**